diagnostic.md

```diff
@@ -6,45 +6,54 @@ Place your responses inside the fenced code-blocks where indicated by comments.
 What are two ways of telling rails where to route a GET request to '/books/4'?
 
 ```rb
-// your response here
+curl --include --request GET http://localhost:4741/books/4
+
+API="${API_ORIGIN:-http://localhost:4741/books/4}"
+URL_PATH="/examples"
+curl "${API}${URL_PATH}" \
+  --include \
+  --request GET \
+  --header "Authorization: Token token=$TOKEN"
+
 ```
 
 In what file does the above code go?
 
 ```md
-// your response here
+This code would go in the scripts folder.  An example file in that folder may be "get-ingredient.sh"
 ```
 
 In your own words, what does the command `bundle exec rake db:migrate` do?
 
 ```md
-// your response here
+It executes the rake script specific to your gemfile.  The command db:migrate runs migrations that have yet to run.
+
```

Comment:
It executes any code found within the Active Record migration files in db/migrate. The migrations make it so that we don't have to modify our database directly.